src/domains/core/Loading/Loading.component.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './styles/loading.scss';
+
+const LoadingComponent = ({ isLoading }) => {
+  const className = isLoading ? 'is-loading' : 'hide-loading';
```

Comment:
The implementation looks odd.

Reply:
```js
const LoadingComponent = ({ isLoading }) => {
  const className = isLoading ? '' : 'hide';
  return <div className={`loading ${className}`} />;
};
```

Reply:
I've read some interesting implementation of stateful styles using the data attribute so the implementation it goes like `<div data-state="loading"></div>` in the css `div[data-state="loading"] { ... }` so you can have more interactions/animations, like handling "idle, loading, success, error", just something to consider, also keeps the classes clean.